client/my-sites/domains/domain-management/components/header/domain-management-header.tsx

```diff
@@ -0,0 +1,155 @@
+import { Circle, SVG } from '@wordpress/components';
+import { home, Icon, info } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import { connect } from 'react-redux';
+import Badge from 'calypso/components/badge';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { isMappedDomain, resolveDomainStatus } from 'calypso/lib/domains';
+import { type as DomainType } from 'calypso/lib/domains/constants';
+import Breadcrumbs from 'calypso/my-sites/domains/domain-management/components/breadcrumbs';
+import { domainManagementList, isUnderDomainManagementAll } from 'calypso/my-sites/domains/paths';
+import getCurrentRoute from 'calypso/state/selectors/get-current-route';
+import isPrimaryDomainBySiteId from 'calypso/state/selectors/is-primary-domain-by-site-id';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import type { DomainManagementHeaderPassedProps, DomainManagementHeaderProps } from './types';
+import type { TranslateResult } from 'i18n-calypso';
+import './style.scss';
+
+const DomainManagementHeader = ( props: DomainManagementHeaderProps ) => {
+	const { selectedDomainName, isManagingAllDomains, selectedSite } = props;
+	const { __ } = useI18n();
+	let formattedHeaderText = selectedDomainName;
+	if ( ! selectedDomainName ) {
+		formattedHeaderText = isManagingAllDomains ? __( 'All Domains' ) : __( 'Site Domains' );
+	}
+
+	const renderCircle = () => (
+		<SVG viewBox="0 0 24 24">
+			<Circle cx="12" cy="12" r="12" />
+		</SVG>
+	);
+
+	const renderBreadcrumbs = () => {
+		const items = [
+			{
+				// translators: Internet domains, e.g. mygroovydomain.com
+				label: __( 'Domains' ),
+				href: domainManagementList( selectedSite!.slug, selectedDomainName ),
+			},
+			{
+				label: selectedDomainName,
+			},
+		];
+
+		const mobileItem = {
+			label: __( 'Back to domains' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Back to All Domains' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-management-header)